### PR TITLE
Added new 404 artwork for GCI

### PR DESCRIPTION
**Pull request by yambati03**

### CODE REVIEW

File: artwork/popup_display/coala404.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...93.82,1193.82,0,0,1,156.52,935.79Z"/><path class="bd91a13c-5aff-4e4d-9af7-c13550ce1439" d="M171,1094.55l23.51.84a719.17,719.17,0,0,0-7.45-131.79l-23.27,3.48A695.15,695.15,0,0,1,171,1094.55Z"/><path class="ca08f01f-5323-43e1-a9e8-ee969e437aca" d="M194.51,1095.39l-11.75-.43-11.76-.42q4.17-31.83,5.4-64.24t-.57-65q7.62,31.74,12.32,64.36T194.51,1095.39Z"/><path class="29239d91-1093-49d3-81ab-0f93cc29e001" d="M176.05,969.67q1.48,30.41.35,60.63-1.22,32.35-5.4,64.24l11.76.42A708.53,708.53,0,0,0,176.05,969.67Z"/><path class="bd91a13c-5aff-4e4d-9af7-c13550ce1439" d="M203.14,1105.71l23.49,1.13,2.37-49.27,1.18-24.63c.4-8.23.77-16.33,1.39-24.28a228.42,228.42,0,0,1,2.86-23.26,137.84,137.84,0,0,1,5.87-22c5.11-14.66,12.53-29.83,17.15-47.25l-22.76-6a196.87,196.87,0,0,1-7.52,22c-2.95,7.5-6.18,15.24-9.06,23.41a162.15,162.15,0,0,0-6.82,25.59,250.83,250.83,0,0,0-3.18,25.67c-.65,8.45-1,16.77-1.43,25l-1.18,24.63Z"/><path class="ca08f01f-5323-43e1-a9e8-ee969e437aca" d="M226.63,1106.84l-11.74-.57-11.75-.57,5.4-49.12,2.7-24.56c.91-8.18,1.79-16.44,2.94-24.72a245.61,245.61,0,0,1,4.6-24.77,152.91,152.91,0,0,1,7.94-23.93c6.48-15.36,14.56-29.61,19.75-45.34-3.29,16.23-9.73,31.71-14.2,47.29a145.82,145.82,0,0,0-4.8
```
Comment (li-boxuan) [inline artwork/popup_display/coala404.svg]: Can you break it down to more lines?
Also missing newline at EOF :)

**Review by srivama — COMMENTED** [review]
Nice idea there, Yashas 👍 The logo, description and 404 aren't centre aligned w.r.to the svg dimensions though. Please make the amendments :)

**Comment by ayan-b** [comment]
Really nice! Next time onwards keep your master branch clean and do your work in a different branch! :smiley:

**Review by KVGarg — CHANGES_REQUESTED** [review]
@yambati03 Commit message isn't in the correct format, have a look at [commit guidelines](http://coala.io/commit) to improve your commit message. 
Please also [rebase](https://coala.io/rebase) ;-)

**Comment by sangamcse** [comment]
ack 628592fc67bcbffc7597331025cc6390e54b5848

**Comment by sangamcse** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward failed! Please fastforward your pull request manually via the command line.

Reason:
```
Command: git push upstream HEAD:master

Exit Code: 1

Cause:

remote: error: GH006: Protected branch update failed for refs/heads/master.        
remote: error: At least 1 approving review is required by reviewers with write access.        
To <hidden_oauth_token>
 ! [remote rejected] HEAD -> master (protected branch hook declined)
error: failed to push some refs to '<hidden_oauth_token>'


```

**Review by sangamcse — APPROVED** [review]

**Comment by sangamcse** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**